14-EventToCommandBehavior/src/PrismSample/Views/EventArgsConverterExamplePage.xaml

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PrismSample.Views.EventArgsConverterExamplePage"
+             Title="Event Args Converter Example"
+             
+             xmlns:c="clr-namespace:PrismSample.Converters"
+             xmlns:b="clr-namespace:Prism.Behaviors;assembly=Prism.Forms">
```

Comment:
this should be `xmlns:prism="http://prismlibrary.com"`

Reply:
I will fix it :)